backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -3,49 +3,110 @@
 import javax.persistence.AttributeOverride;
 import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.OneToOne;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 
+import gov.cdc.usds.simplereport.db.model.auxiliary.AskOnEntrySurvey;
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestCorrectionStatus;
 import org.hibernate.annotations.Immutable;
 import org.hibernate.annotations.Type;
 
 import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Date;
+import java.util.UUID;
 
 @Entity
 @Immutable
 @AttributeOverride(name = "result", column = @Column(nullable = false))
 public class TestEvent extends BaseTestInfo {
+	private static final Logger LOG = LoggerFactory.getLogger(TestEvent.class);
 
 	@Column
 	@Type(type = "jsonb")
 	private Person patientData;
+
 	@Column
 	@Type(type = "jsonb")
 	private Provider providerData;
-	@OneToOne(mappedBy = "testEvent")
+
+	@Column
+	@Type(type = "jsonb")
+	private AskOnEntrySurvey patientAnswersData;
```

Comment:
👍 good call—no sense managing multiple rows of a table that just contains a JSON object, when we could just copy the JSON object. I think that side table may end up being a mistake, since we're adding so much to TestEvent and TestOrder anyway...

Reply:
However, I think we should call it `survey` or `askOnEntry`, not `patientAnswersData` — it's (correctly!) typed as an AskOnEntrySurvey, not the (mostly useless, as it turns out) PatientAnswers container type.